notebooks/en/agents.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<p>&gt;maybe give a link to transformers agents</p><p>&gt;enable it: enable the system </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thank you, added the link and your wording correction!